pkg/asset/machines/gcp/machines.go

```diff
@@ -74,6 +75,14 @@ func provider(clusterID string, platform *gcp.Platform, mpool *gcp.MachinePool,
 		return nil, err
 	}
 
+	if mpool.OSDisk.DiskType == "" {
+		mpool.OSDisk.DiskType = "pd-ssd"
+	}
+
+	if mpool.OSDisk.DiskSizeGB == 0 {
+		mpool.OSDisk.DiskSizeGB = 128
+	}
```

Comment:
You should set the defaults in the default machine pool: https://github.com/openshift/installer/blob/master/pkg/asset/machines/worker.go#L95-L99